layers/meta-balena-beaglebone/conf/layer.conf

```diff
@@ -26,9 +26,23 @@ PREFERRED_PROVIDER_u-boot:beagleboard-xm = "u-boot"
 KERNEL_IMAGETYPES:beaglebone-ai64 = "Image"
 KERNEL_CLASSES:beaglebone-ai64 = "kernel-uimage"
 
+PREFERRED_PROVIDER_virtual/kernel:beagleplay = "linux-bb.org"
+PREFERRED_PROVIDER_virtual/bootloader:beagleplay = "u-boot-ti-staging"
+PREFERRED_PROVIDER_u-boot:beagleplay = "u-boot-ti-staging"
+PREFERRED_VERSION_u-boot-tools:beagleplay ?= "2022.01"
+PREFERRED_VERSION_u-boot-tools-native:beagleplay ?= "2022.01"
+
+PREFERRED_PROVIDER_virtual/kernel:beagleplay-k3r5 = "linux-bb.org"
+PREFERRED_PROVIDER_virtual/bootloader:beagleplay-k3r5 = "u-boot-ti-staging"
```

Comment:
This can be removed as per the Zulip discussion?

Reply:
Again, see my response above.